app/src/main/res/values/strings.xml

```diff
@@ -22,4 +22,5 @@
     <string name="action_ok">Got it!</string>
     <string name="error_could_not_set_current_location">Could not set memory location to current.</string>
     <string name="action_apply_theme">Apply Theme</string>
+    <string name="action_change_memory_location">Change memory location</string>
```

Comment:
I think this should be changed from `Change memory location` to `Change location`

Reply:
done